### PR TITLE
ITEM.tsv consistency check

### PR DESCRIPTION
**Pull request by JayShee**
Consistency check for ITEM.tsv

### CODE REVIEW

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -1874,11 +1874,11 @@ ITEM_20150317_001873	iCoin Box
 ITEM_20150317_001874	Contains 1 iCoin. Right-click to use. 
 ITEM_20150317_001875	Cures the [Poison] status ailment. 
 ITEM_20150317_001876	Velheider's Food 
-ITEM_20150317_001877	Velhider's favorite food. Completely restores your companion's HP and restores their stamina by 30. 
+ITEM_20150317_001877	Velhider's favorite food. Completely restores your companion's HP and restores their Stamina by 30.
```
Comment (TentuZero) [inline EnglishTranslation-master/ITEM.tsv]: You forgot the e. it's Velheider.

File: EnglishTranslation-master/ITEM.tsv [hunk 2]
```diff
@@ -8749,51 +8749,51 @@ ITEM_20151224_008748	Handle
...
-ITEM_20151224_008789	Earth Resistance Potion
-ITEM_20151224_008790	Increases earth resistance by 30 for 1 minute. Right-click to use. Cooldown of 90 seconds.
-ITEM_20151224_008791	Lightning Resistance Potion
-ITEM_20151224_008792	Increases lightning resistance by 30 for 1 minute. Right-click to use. Cooldown of 90 seconds.
+ITEM_20151224_008782	Increases your critical attack by 80 for 120 seconds. {nl}Right-click to use. Cooldown of 60 seconds.
+ITEM_20151224_008783	Fire Property Resistance Potion
+ITEM_20151224_008784	Increases your Fire property resistance by 30 for 1 minute. {nl}Right-click to use. Cooldown of 90 seconds.
+ITEM_20151224_008785	Ice Property Resistance Potion
+ITEM_20151224_008786	Increases your Ice property resistance by 30 for 1 minute. {nl}Right-click to use. Cooldown of 90 seconds.
+ITEM_20151224_008787	Posion Property Resistance Potion
```
Comment (TentuZero) [inline EnglishTranslation-master/ITEM.tsv]: Poison.

File: EnglishTranslation-master/ITEM.tsv [hunk 3]
```diff
@@ -6193,31 +6193,31 @@ ITEM_20150918_006192	Tyla Plate Armor
...
+ITEM_20150918_006198	Instantly restores 640 HP. {nl}Additionally restores HP every 2 seconds. {nl}896 HP is restored over 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds.
+ITEM_20150918_006199	Instantly restores 70 SP. {nl}Additionally restores SP every 2 seconds. {nl}98 HP is restored over 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds.
+ITEM_20150918_006200	Instantly restores 120 SP. {nl}Additionally restores SP every 2 seconds. {nl}168 HP is restored over 15 seconds. {nl}Right-click to use. Cooldown of 45 seconds.
+ITEM_20150918_006201	Instantly restores 240 SP. {nl}Additionally restores SP every 2 seconds. {nl}336 HP is restored over 15 seconds. {nl}Right-click to use. Cooldown of 60 seconds.
 ITEM_20150918_006202	Small Stamina Pill
 ITEM_20150918_006203	Restores Stamina by 40. Right-click to use. Cooldown of 45 seconds.
 ITEM_20150918_006204	Big Stamina Pill
 ITEM_20150918_006205	Restores Stamina by 40. Right-click to use. Cooldown of 60 seconds.
 ITEM_20150918_006206	Many scholars have tried to find out the origins of this fruit, but none have succeeded this far.
```
Comment (TentuZero) [inline EnglishTranslation-master/ITEM.tsv]: Shouldn't that say, thus far?

Comment (TentuZero) [inline EnglishTranslation-master/ITEM.tsv]: Shouldn't that say, thus far?

**Comment by Sunaries** [comment]
Good to merge. :+1: Nice.

**Comment by JayShee** [comment]
Don't merge yet, checking few more stuff.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**